packages/react-components/src/components/gantt-chart-output-component.tsx

```diff
@@ -0,0 +1,1712 @@
+            document.getElementById(this.props.traceId + this.props.outputDescriptor.id + 'focusContainer')?.focus();
+        } else {
+            document.getElementById(this.props.traceId + this.props.outputDescriptor.id)?.focus();
+        }
+    }
+
+    protected getVerticalScrollbar(): JSX.Element {
+        return (
+            <ReactTimeGraphContainer
+                id="vscroll"
+                options={{
+                    id: 'vscroll',
+                    width: 10,
+                    height: parseInt(this.props.style.height.toString()),
+                    backgroundColor: this.props.style.naviBackgroundColor
+                }}
+                addWidgetResizeHandler={this.props.addWidgetResizeHandler}
+                removeWidgetResizeHandler={this.props.removeWidgetResizeHandler}
+                unitController={this.props.unitController}
+                layers={[this.vscrollLayer]}
+            ></ReactTimeGraphContainer>
+        );
+    }
+
+    private async onElementSelected(element: TimeGraphStateComponent | undefined) {
+        let tooltipObject = undefined;
+        if (element && this.props.viewRange) {
+            tooltipObject = await this.fetchTooltip(element);
+        }
+        if (tooltipObject) {
+            signalManager().emit(
+                'ITEM_PROPERTIES_UPDATED',
+                new ItemPropertiesSignalPayload(tooltipObject, this.props.traceId, this.props.outputDescriptor.id)
+            );
+        }
+    }
+
+    private getTimegraphRowIds() {
+        const { ganttChartTree: timegraphTree, columns, collapsedNodes } = this.state;
+        const rowIds = getAllExpandedNodeIds(listToTree(timegraphTree, columns), collapsedNodes);
+        return { rowIds };
+    }
+
+    private async fetchGanttChartData(
+        range: TimelineChart.TimeGraphRange,
+        resolution: number,
+        fetchArrows: boolean,
+        rowIds?: number[],
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        additionalProperties?: { [key: string]: any }
+    ) {
+        if (document.getElementById(this.props.traceId + this.props.outputDescriptor.id + 'handleSpinner')) {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            document.getElementById(
+                this.props.traceId + this.props.outputDescriptor.id + 'handleSpinner'
+            )!.style.visibility = 'visible';
+        }
+
+        const strategy = additionalProperties?.filter_query_parameters?.strategy;
+        const ids = rowIds ? rowIds : this.getTimegraphRowIds().rowIds;
+        const { start, end } = range;
+        const newRange: TimelineChart.TimeGraphRange = range;
+        const nbTimes = Math.ceil(Number(end - start) / resolution) + 1;
+        const timeGraphData: TimelineChart.TimeGraphModel = await this.tspDataProvider.getData(
+            ids,
+            this.state.ganttChartTree,
+            fetchArrows,
```

Comment:
can we set it to false for ganttCharts by default?

Reply:
Not sure I understand which attribute are you referring to?